test/conformance/tests/test_sriov_operator.go

```diff
@@ -826,10 +820,7 @@ var _ = Describe("[sriov] operator", func() {
 				ipam := ipamIpv4
 				err = network.CreateSriovNetwork(clients, sriovDevice, sriovNetworkName, ns1, operatorNamespace, resourceName, ipam)
 				Expect(err).ToNot(HaveOccurred())
-				Eventually(func() error {
-					netAttDef := &netattdefv1.NetworkAttachmentDefinition{}
-					return clients.Get(context.Background(), runtimeclient.ObjectKey{Name: sriovNetworkName, Namespace: ns1}, netAttDef)
-				}, (30+snoTimeoutMultiplier*90)*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
```

Comment:
two comments:

1. can we call waitForNetAttachDef(...) in L841 as well ?
2. we now wait 10 seconds and not 30 when muliplier is 0 , if it doesnt fail i guess that is ok.

Reply:
> two comments:
> 
> 1. can we call waitForNetAttachDef(...) in L841 as well ?

sure! didn't see it

> 2. we now wait 10 seconds and not 30 when muliplier is 0 , if it doesnt fail i guess that is ok.

I found no reason for having 30s instead of 10s:

- https://github.com/k8snetworkplumbingwg/sriov-network-operator/commit/bed21a02ad29338517794819a6c6dd0e4a69b4e7#diff-8841b252b9d2e29927af1036582b38ac8466067019d67dd5a59f22285674310dR507
- https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/59/commits/0cfd998b2e0b7ae2f2778e522e21e66dbdff5bef#diff-8841b252b9d2e29927af1036582b38ac8466067019d67dd5a59f22285674310dR642

I'd go with this and see if it flake.